samples/sample-manifest.json

```diff
@@ -20,13 +20,32 @@
             "MaxActivations": 1,
             "Parameters": {
                 "Files": {
-                    "/tmp/defg.txt": "foo",
-                    "/tmp/jkl.mno": "bar"
+                    "/tmp/abc.txt": "helloworld",
+                    "/tmp/helloworld.sh": "#!/usr/bin/env bash\necho {{ string .Secrets.hello }} {{ string .Secrets.hello }}",
+                    "/tmp/defg.txt": {
+                        "Encoding": "string",
+                        "NoTemplates": false,
+                        "Data": "foo"
+                    },
+                    "/tmp/jkl.mno": {
+                        "Encoding": "base64",
+                        "NoTemplates": true,
+                        "Data": "YmFy"
+                    },
+                    "/tmp/enc.key": {
+                        "Encoding": "string",
+                        "NoTemplates": true,
+                        "Data": "{{ string .Secrets.symmetric_key_shared }}"
+                    }
                 },
                 "Env": {
                     "IS_FIRST": "true",
+                    "IS_SECOND": {
+                        "Encoding": "string",
+                        "Data": "false"
+                    },
                     "SEAL_KEY": "{{ hex .Marblerun.SealKey }}",
-                    "TEST_SECRET_SYMMETRIC_KEY": "{{ raw .Secrets.symmetric_key_shared }}",
+                    "API_KEY": "{{ string .Secrets.api_key }}",
```

Comment:
```suggestion
                    "API_KEY": "{{ string .Secrets.apiKey }}",
```

Reply:
Just noticed that the original proposal included the renaming of `raw` to `string`.
Are we set on this? Or is this still open to discussion?
What are your thoughts on this @Nirusu ?

Reply:
I think keeping `raw` for now is fine. There is too much confusion going on already.